org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/TestProjectDescriptor.xtend

```diff
@@ -44,14 +45,58 @@ abstract class TestProjectDescriptor extends ProjectDescriptor {
 	override getExternalDependencies() {
 		val deps = newLinkedHashSet
 		deps += super.externalDependencies
-		deps += new ExternalDependency()=>[
-			p2.bundleId = "org.junit"
-			p2.version = "4.12.0"
-			maven.groupId = "junit"
-			maven.artifactId = "junit"
-			maven.version = "4.12"
-			maven.scope = Scope.TESTCOMPILE
-		]
+		if (config.junitVersion == JUnitVersion.JUNIT_4) {
+			deps += new ExternalDependency()=>[
+				p2.bundleId = "org.junit"
+				p2.version = "4.12.0"
+				maven.groupId = "junit"
+				maven.artifactId = "junit"
+				maven.version = "4.12"
+				maven.scope = Scope.TESTCOMPILE
+			]
+		}
+		if (config.junitVersion == JUnitVersion.JUNIT_5) {
+			deps += new ExternalDependency()=>[
+				p2.bundleId = "org.junit.jupiter.api"
+				p2.version = "5.1.0"
```

Comment:
This appears to be inconsistent with the 5.0.0 in the manifest

Reply:
OK, will use 5.1.0 consistent